doc/source/whatsnew/v0.18.2.txt

```diff
@@ -101,6 +101,7 @@ API changes
 ~~~~~~~~~~~
 
 
+- ``DataFrame.to_csv()`` has dropped the ``engine`` parameter (:issue:`13418`)
```

Comment:
True, though the ignoring warning and breaking all of my code argument could be made for a major release as well...but fair enough: will break this up.